File: gemini_tests/java/CommandInjection.java

import java.io.IOException;
import javax.servlet.http.HttpServletRequest;

public class CommandInjection {
    public void vulnerable(HttpServletRequest request) throws IOException {
        String command = request.getParameter("command");
        // Vulnerable to Command Injection
        Runtime.getRuntime().exec("cmd.exe /c " + command);
    }
}

File: smoke_tests/go/command_injection.go

package main

import (
	"net/http"
	"os/exec"
)

func vulnerable(w http.ResponseWriter, r *http.Request) {
	cmdStr := r.URL.Query().Get("cmd")
	// Vulnerable to Command Injection
	cmd := exec.Command(cmdStr)
	cmd.Run()
}
